### ISSUE TITLE
optimizing the neat with backpropagation ?

### ISSUE THREAD

**Issue by winatawelly** [crisbodnar/TensorFlow-NEAT#2]
do you think it is possible to backprop the neat result ? if so what do you think the best way(s) to do that ?
thanks!

**Comment by crisbodnar** [comment]
Yes. As long as the evolved network is differentiable, this can be done. This is not different in any way from optimising a regular neural network, so you should find enough resources online for doing this.
